hedera-node/src/main/java/com/hedera/services/stats/HapiOpCounters.java

```diff
@@ -70,21 +73,24 @@ public HapiOpCounters(
 		Arrays.stream(allFunctions.get())
 				.filter(function -> !IGNORED_FUNCTIONS.contains(function))
 				.forEach(function -> {
-			receivedOps.put(function, new AtomicLong());
-			if (QUERY_FUNCTIONS.contains(function)) {
-				answeredQueries.put(function, new AtomicLong());
-			} else {
-				submittedTxns.put(function, new AtomicLong());
-				handledTxns.put(function, new AtomicLong());
-			}
-		});
+					receivedOps.put(function, new AtomicLong());
+					if (QUERY_FUNCTIONS.contains(function)) {
+						answeredQueries.put(function, new AtomicLong());
+					} else {
+						submittedTxns.put(function, new AtomicLong());
+						handledTxns.put(function, new AtomicLong());
+					}
+				});
+		receivedDeprecatedTxns = new AtomicLong();
 	}
 
 	public void registerWith(final Platform platform) {
 		registerCounters(platform, receivedOps, COUNTER_RECEIVED_NAME_TPL, COUNTER_RECEIVED_DESC_TPL);
 		registerCounters(platform, submittedTxns, COUNTER_SUBMITTED_NAME_TPL, COUNTER_SUBMITTED_DESC_TPL);
 		registerCounters(platform, handledTxns, COUNTER_HANDLED_NAME_TPL, COUNTER_HANDLED_DESC_TPL);
 		registerCounters(platform, answeredQueries, COUNTER_ANSWERED_NAME_TPL, COUNTER_ANSWERED_DESC_TPL);
+
+		platform.addAppStatEntry(counter.from(COUNTER_RECEIVED_DEPRECATED_NAME_TPL, COUNTER_RECEIVED_DEPRECATED_DESC_TPL, receivedDeprecatedTxns::get));
```

Comment:
nit : easier to read if we had a method `registerCounter` similar to `registerCounters` that will take just an `AtomicLong` instead of a map and register the counter `receivedDeprecatedTxns` with platform

Reply:
yes, as it was only one counter, did not do it. Made the changes now. Thanks @anighanta